pycbc/derived_parameters.py

```diff
@@ -0,0 +1,462 @@
+# Copyright (C) 2017  Collin Capano
```

Comment:
Module can be named a bit shorter. How about just 'conversions' or something even shorter. @cmbiwer Ideas?